dfastmi/gui/dialog_view.py

```diff
@@ -224,17 +226,21 @@ def _update_ucritical(self, ucrit: float, default: float):
             gui_text("ucrit", placeholder_dictionary={"default": str(default)})
         )
 
-    def _update_qthreshold(self, data: float):
+    def _update_qthreshold(self, qthreshold: float, qstagnant: float):
         """
         Update the GUI components when the discharge threshold changes.
 
         Args:
-            data: The dischage threshold.
+            qthreshold: The dischage threshold.
+            qstagnant: The dischage stagnant threshold.
         """
         # Update the threshold discharge in the GUI
-        self._qthr.setText(str(data))
+        self._qthr.setText(str(qthreshold))
 
         # Update labels and text fields
+        self._qthrtxt.setText(
```

Comment:
Maybe give gthrtxt a different name, now its a little bit confusing with the _qthr.setText and the _qthrtxt.setText

The _qthr is the input field right?

and the _qthrtxt is the label infront of the input field which is updated with the stagnant value right?

Reply:
yes. but was already name locally like this. To avoid confision will rename to _qtrlabel